src/index.js

```diff
@@ -9,6 +9,7 @@ import installHooks from './lib/output-hooks';
 import pkg from '../package.json';
 import logo from './lib/logo';
 import checkVersion from './../check';
+import updateNotifier from 'update-notifier';
```

Comment:
Let's move this to line 2, so that it precedes all the local imports.

Reply:
Done. Didn't think about this earlier.